<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url('../Images/background.png');
background-repeat: no-repeat;
background-position: center;
background-size: cover; 
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="hive_label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>150</y>
      <width>491</width>
      <height>198</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>128</pointsize>
     </font>
    </property>
    <property name="text">
     <string>HIVE</string>
    </property>
   </widget>
   <widget class="QLabel" name="hive_photo">
    <property name="geometry">
     <rect>
      <x>808</x>
      <y>98</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../Images/hive_photo.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="game_label">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>140</y>
      <width>561</width>
      <height>198</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>128</pointsize>
     </font>
    </property>
    <property name="text">
     <string>GAME</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_game_button">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>482</y>
      <width>523</width>
      <height>99</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_button">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>692</y>
      <width>523</width>
      <height>99</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
